core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -175,29 +183,42 @@ void forceRefresh() {
    * would expire.
    */
   private InstanceData performRefresh() throws InterruptedException, ExecutionException {
+    logger.fine("Refresh Operation: Acquiring rate limiter permit");
     // To avoid unreasonable SQL Admin API usage, use a rate limit to throttle our usage.
     forcedRenewRateLimiter.acquirePermit();
+    logger.fine("Refresh Operation: Starting...");
```

Comment:
Change the messages to parallel before and after blocking operation: 
- "Refresh Operation: Acquiring rate limiter permit..."
- "Refresh Operation: Acquired rate limiter permit. Starting refresh..."
- "Refresh Operation: Completed refresh, new certificate expiration at %s."
- "Refresh Operation: Next operation scheduled at %s."

Reply:
Done.